docs/gethue/content/en/posts/2021-07-26-zero-click-hue-importer-drag-and-drop-a-file-auto-create-a-table.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Zero Click Hue Importer -- Drag and drop a file, auto create a table
+author: Hue Team
+type: post
+date: 2021-07-26T00:00:00+00:00
+url: /blog/2021-07-26-zero-click-hue-importer-drag-and-drop-a-file-auto-create-a-table
+sf_thumbnail_type:
+  - none
+sf_thumbnail_link_type:
+  - link_to_post
+sf_detail_type:
+  - none
+sf_page_title:
+  - 1
+sf_page_title_style:
+  - standard
+sf_no_breadcrumbs:
+  - 1
+sf_page_title_bg:
+  - none
+sf_page_title_text_style:
+  - light
+sf_background_image_size:
+  - cover
+sf_social_sharing:
+  - 1
+sf_related_articles:
+  - 1
+sf_sidebar_config:
+  - left-sidebar
+sf_left_sidebar:
+  - Sidebar-2
+sf_right_sidebar:
+  - Sidebar-1
+sf_caption_position:
+  - caption-right
+sf_remove_promo_bar:
+  - 1
+ampforwp-amp-on-off:
+  - default
+categories:
+  - Version 4.11
+  - Development
+  - Query
+
+---
+
+In the last month, we introduced [creating a table from the local file](/blog/2021-05-26-improved-hue-importer-select-a-file-choose-a-dialect-create-a-table/) in importer. In that feature, if you want to create a table then you must have to follow the steps provided in that blog post but now only drag and drop a CSV file in the left assist, then it automatically creates the table for you.
```

Comment:
`In that feature, if you want to create a table then you must have to follow the steps provided in that blog post but now only drag and drop a CSV file in the left assist, then it automatically creates the table for you.
Write` no clear. 

must have to follow is no very proper

e.g.

Creating a new table previously required you to setup a storage account (e.g. HDFS, S3, ADLS...) in order to upload the data. The list of SQL dialects was also restrictive with no support for Apache Phoenix (blog link to add..).

Now creating a table is even easier as simply drag & dropping a CSV file on the table listing will perform it!

Reply:
+1.
Previously, creating a new table required you to set up a storage account such as HDFS, S3, or ADLS to upload data. The list of SQL dialects was also restrictive, with no support for Apache Phoenix. Now, creating a table is even easier. Just drag and drop a CSV file on the table listing, and Hue automatically creates a SQL table for you!

Reply:
Done thanks @pychheda @romainr